packages/waku/src/types.d.ts

```diff
@@ -3,11 +3,3 @@ declare module "react-server-dom-webpack/server";
 declare module "react-server-dom-webpack/server.node.unbundled";
 declare module "react-server-dom-webpack/client";
 declare module "react-server-dom-webpack/client.node.unbundled";
-
-// FIXME Is this too naive? How can we avoid the eslint warning?
```

Comment:
Nice! How did you do it? Which change does this job?

Reply:
`react/experimental`

Reply:
Okay, yeah, I'd love to have in the source code instead of tsconfig.